core-test/src/test/scala/org/bitcoins/core/crypto/AesCryptTest.scala

```diff
@@ -0,0 +1,58 @@
+package org.bitcoins.core.crypto
+
+import org.bitcoins.testkit.core.gen.{NumberGenerator, StringGenerators}
+import org.bitcoins.testkit.util.BitcoinSUnitTest
+import scodec.bits.HexStringSyntax
+
+class AesCryptTest extends BitcoinSUnitTest {
+  behavior of "AesEncrypt"
+
+  val password = AesPassword("PASSWORD")
+  val badPassword = AesPassword("BAD_PASSWORD")
+  val emptyPassword = AesPassword("")
+
```

Comment:
We are missing a couple of hard coded test vectors with expected ecrypted output and then the corresponding decrypted output. These are really useful for sanity tests. Property tests can be misleading some times as you can implement the encrypt and decrypt functionality in just the right way so that your tests pass :-)